recipes/xsarsea/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "xsarsea" %}
+{% set version = "0.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xsarsea-{{ version }}.tar.gz
+  sha256: 454e07b312d79f86d2afba05c7cd95cf26e7a406dd852d2c177be3d30db6607f
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

Comment:
This one has scripts and cannot be noarch but if upstream abandon the deprecated script, we can change this later in the feedstock.

Reply:
Ok to change this. Thanks for the merge.